app/main.py

```diff
@@ -1,56 +1,44 @@
-#from typing import List
-#from uuid import UUID
+from typing import List
+# from uuid import UUID
+import uvicorn
 from fastapi import FastAPI, HTTPException
-#from pymongo import MongoClient
+from pydantic import BaseModel
+from pymongo import MongoClient
 
-#from models.models import Gender, Role, User
+# from models.models import Gender, Role, User
 
 app = FastAPI()
 
-#client = MongoClient("mongodb://localhost:27017/")
-#db = client["GovernmentCatnip"]
-
-#db: List[User] = [
-#    User(
-#        id="31b6e462-4c9a-4d3d-a00a-e7a8fc43c4e7", 
-#        first_name="Jamila", 
-#        last_name="Ahmed",
-#        gender=Gender.female,
-#        roles=[Role.student]
-#    ),
-#    User(
-#        id="03dfb08b-faa6-47fc-b60c-3484a62f8b2a", 
-#        first_name="Alex", 
-#        last_name="Jones",
-#        gender=Gender.male,
-#        roles=[Role.admin, Role.user]
-#    )
-#]
+client = MongoClient(host="db")
+db = client["government_catnip"]
+db_populations = db["personal_information"]
 
 @app.get("/")
 async def root():
     return {
         "message": "Sup for our documentation go to link variable",
         "link": "https://catnip-govenment-module.github.io/government-catnip"
     }
-    
-#@app.get("/api/v1/users")
-#async def fetch_users():
-#    return db;
-#
-#@app.post("/api/v1/users")
-#async def register_user(user: User):
-#    db.append(user)
-#    return {"id": user.id}
-#
-#@app.delete("/api/v1/users/{user_id}")
-#async def delete_user(user_id: UUID):
-#        for user in db:
-#            if user.id == user_id:
-#                db.remove(user)
-#                return
-#        raise HTTPException(
-#            status_code=404,
-#            detail=f"user with id: {user_id} does not exists"
-#        )
-#
\ No newline at end of file
+
+
+class Population(BaseModel):
+    _id: int
+    title: str
+    firstName: str
+    lastName: str
+    sex: str
+    locationID: int
+    rightToVote: bool
+    blackList: bool
+
+class Population_p(BaseModel):
+    detail: List[Population]

```

Comment:
Can we move the model to another directory like ```app/models/``` so that the code appears cleaner

Reply:
Thanks! fixed this